yarn/src/main/scala/org/apache/spark/deploy/yarn/YarnAllocator.scala

```diff
@@ -21,9 +21,10 @@ import java.util.Collections
 import java.util.concurrent._
 import java.util.regex.Pattern
 
+import scala.collection.JavaConverters._
 import scala.collection.mutable
 import scala.collection.mutable.{ArrayBuffer, HashMap, HashSet}
-import scala.collection.JavaConverters._
+import scala.collection.immutable.Set
```

Comment:
super nit: I think JavaConverters was actually correct in the old location (order is not just alphabetic -- full classes sort before packages).  Also immutable should go before mutable.